### ISSUE TITLE
Separate effect design tools from environment tools.

### ISSUE THREAD

**Issue by camparijet** [openpool/openpool-core#16]
Now effects designers are required the same environment as the system glued to a billiard table.

This raises two bad points for designers :
1. setup processes include some platform dependency like issues #14      https://github.com/openpool/openpool-base-kinect/issues/14
2. after complex setup process, you'll find there is a big diffrence between your machine and running system.

**Comment by takashyx** [comment]
Agree, maybe we need something like "one-click install" simulator package for the creators.
